apps/etterlatte-beregning/src/main/kotlin/beregning/BeregningService.kt

```diff
@@ -93,6 +95,23 @@ class BeregningService(
         return beregningRepository.hentOverstyrBeregning(behandling.sak)
     }
 
+    suspend fun opprettOverstyrBeregning(
+        behandlingId: UUID,
+        overstyrBeregning: OverstyrBeregningDTO,
+        brukerTokenInfo: BrukerTokenInfo,
+    ): OverstyrBeregning? {
+        val behandling = behandlingKlient.hentBehandling(behandlingId, brukerTokenInfo)
+
+        return hentOverstyrBeregning(behandlingId, brukerTokenInfo).takeIf { it != null }
```

Comment:
Blir dette riktig viss du endrar på ei allereie overstyrt beregning?

Eller er det kanskje ikkje noko vi opnar for i frontend i det heile tatt?

Reply:
Og no ser eg svaret på spørsmålet mitt i ein av testane under her, ja. Fine greier.

Reply:
Dette er for migrering. Hvis en sak er satt til manuelt så er den satt - vi har ikke støtte for å fjerne - så hvis det kaller to ganger - så får du tilbake at det er satt - men du får opprinnelig melding tilbake. Vi støtter ikke endring.